tests/unit/moduleapi/async_rm_call.tcl

```diff
@@ -374,8 +376,11 @@ start_server {tags {"modules"}} {
             {del string_foo}
             {set string_foo 1}
             {set string_bar 2}
+            {incr before_deleted}
+            {incr before_expired}
```

Comment:
it's a little hard for me to keep track of this, maybe we can include the key name in the output?
IIRC we can't comment inside this list.
maybe no change is needed / possible, but can you at least explain to me what's going on here?

Reply:
Yes we can not comment inside, maybe I can comment above?
The first before_deleted is on the list itself that gets empty.
The second is also on the list itself, after that we have the lazy expiration of  `string_foo` (before_expired). The `before_overwritten` is because we reset `string_bar` key.

Reply:
maybe instead of `incr before_expired` it could be `lpush before_expired string_foo`?
i.e. IIRC we started using counters so we can see how many events were propagated, but this way we can see what exactly each one did (and we can still count them).
WDYT?
maybe not worth bothering about it at this stage though.